src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -845,7 +845,6 @@ public function providePathsForMakePathRelative()
             array('/var/lib/symfony/src/Symfony/', '/var/lib/symfony/src/Symfony/Component/', '../'),
```

Comment:
Yes, this does work with my fixes. But if relative paths should be supported at all is currently in discussion as you can see in the comments of this pull request.